manager/integration/tests/test_cloning.py

```diff
@@ -289,6 +289,64 @@ def test_cloning_with_backing_image():  # NOQA
     3. Clean up the test
     """
 
+    # Create storage class with backing image
```

Comment:
I believe this test is the same as `test_cloning_basic` except the storage class is different. 

I would suggest reusing `test_cloning_basic` w/ some changes by providing a storage class parameter of backing image instead of repeating the similar steps again?

Reply:
Hi @innobead ,

Yes this case is from `test_cloning_basic`, I will modify code to use `test_cloning_basic` to achieve test goal